### ISSUE TITLE
Rule: Disallow extraneous white lines.

### ISSUE THREAD

**Issue by martndemus** [ember-template-lint/ember-cli-template-lint#38]
``` hbs
<h1>Foo</h1>


<ul>
  <li>something</li>
  <li>interesting</li>
</ul>
```

Should warn about an extra white line on line 3.

**Comment by rwjblue** [comment]
As of v0.3.4 (from HTMLBars 0.4.17) `TextNode`'s now have location information.  Which may make this easier...

**Comment by Turbo87** [comment]
moved to https://github.com/ember-template-lint/ember-template-lint/issues/385
